10/umbraco-deploy/release-notes.md

```diff
@@ -130,74 +173,163 @@ We've listed here all changes going back to March 2021 for Deploy 4 and above. F
+*   Improved UX and descriptions in backoffice settings dashboard [#118](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/118)
+*   Added ability to download Deploy artifacts (.uda files) as a zip archive from the management dashboard.
+*   Added sort options to the schema comparison view in the management dashboard [#115](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/115)
+*   Indented the JSON representation of Data Type configuration details in the .uda files for ease of review [#85](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/85)
+*   Fixed issue with transfer of Forms prevalue sources from text files that include captions.
+*   Ensured Document Type validation messages are transferred between environments [#137](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/137)
+
+
+
+### [Deploy 4.6.2](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.2) (August 16th 2022)
+
+*   Aligned Git URL displayed in backoffice with that in Cloud Portal (V4 only) [#136](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/136)
+*   Fixed issue with deployment of root node value for Umbraco Forms's "save as Umbraco node" workflow [#133](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/133)
+*   Fixed incorrect availability of workspace restore in production environment (V9 and V10 only)
+*   Added close button to "Transfer now" dialog
+*   Resolved registration of deployable types to support configuration for "backoffice edition".
+
+### [Deploy 4.6.1,](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.1) (July 12th 2022)
+
+*   Resolved issue with media restore when database items exist and files don't [#123](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/123)
+*   Added details of failed deployment to deploy dashboard [#120](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/120)
+*   Added copy button for deploy log in the backoffice [#121](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/121)
+*   Fixed typo in UI [#113](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/113)
+*   Ensured signature refresh on Data Type move into or out of folder [#125](https://github.com/umbraco/Umbraco.Deploy.Issues/issues/125)
+*   Fixed selection of workspace for compare dialog
+*   Optimized existence checks in connectors
+*   Restore missing partial restore option in content and media tree roots (V9+)
+*   Fixed extract trigger URL in PowerShell script distributed with Deploy On-Premise (V9+)
+*   Improved deserialization of exceptions for clearer reporting (V9+)
+
+
+### [Deploy 4.6.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.0) [and](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.12.0)  (April 26th 2022)
```

Comment:
```suggestion
### [Deploy 4.6.0](https://github.com/umbraco/Umbraco.Deploy.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F4.6.0) [and] [Deploy 8.12.0](https://github.com/umbraco/Umbraco.Forms.Issues/issues?q=is%3Aissue+is%3Aclosed+label%3Arelease%2F8.12.0)  (April 26th 2022)
```

Reply:
Thanks for the suggestion :D It seems it is a link to a forms issue, so I'll just remove it :D